crates/rolldown_binding/src/options/binding_input_options/binding_watch_option.rs

```diff
@@ -6,7 +6,7 @@ use crate::options::plugin::types::binding_js_or_regex::{
   bindingify_string_or_regex_array, BindingStringOrRegex,
 };
 
-#[napi_derive::napi(object)]
+#[napi_derive::napi(object, object_to_js = false)]
```

Comment:
Why need `object_to_js`  here?
https://napi.rs/changelog/napi_derive#disable-tonapivalue-for-object

Reply:
This is because the current `BindingStringOrRegex` only implements the `FromNapiValue` trait and does not implement `ToNapiValue`. At present, it seems that `object_to_js` is unnecessary. If we need it in the future, we can implement it again, but there may be some performance trade-offs.